pkg/reconciler/configuration/configuration.go

```diff
@@ -59,15 +59,20 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, config *v1.Configuration
 	recorder := controller.GetEventRecorder(ctx)
 
 	// First, fetch the revision that should exist for the current generation.
-	lcr, err := c.latestCreatedRevision(config)
+	lcr, err := c.latestCreatedRevision(ctx, config)
 	if errors.IsNotFound(err) {
 		lcr, err = c.createRevision(ctx, config)
 		if err != nil {
 			recorder.Eventf(config, corev1.EventTypeWarning, "CreationFailed", "Failed to create Revision: %v", err)
```

Comment:
Just to check: do we still want to emit this error if the revision was created but we racily tried it again and failed? Seems potentially a bit confusing

Reply:
Ehr yeah, let's move into the if block